packages/suite/src/views/wallet/coinmarket/sell/offers/index.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { useSelector } from '@suite-hooks';
+import { WalletLayout } from '@wallet-components';
+import styled from 'styled-components';
+import { Props } from '@wallet-types/coinmarketSellOffers';
+import { CoinmarketSellOffersContext, useOffers } from '@wallet-hooks/useCoinmarketSellOffers';
+import Offers from './Offers';
+
+const Wrapper = styled.div`
+    display: flex;
+    width: 100%;
+    flex-direction: column;
+`;
+
+const OffersIndexLoaded = (props: Props) => {
+    const { selectedAccount } = props;
+    const coinmarketOffersValues = useOffers({ ...props, selectedAccount });
```

Comment:
```suggestion
    const coinmarketOffersValues = useOffers(props);
```
is spreading necessary?

Reply:
done